tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -21,50 +20,55 @@ def create_bash_step(script: str) -> ForwardModel:
     )
 
 
-def create_jobs_json(path: Path, steps: Sequence[ForwardModel]) -> None:
+def create_jobs_json(realization: Realization) -> None:
     jobs = {
         "global_environment": {},
         "config_path": "/dev/null",
         "config_file": "/dev/null",
         "jobList": [
             {
-                "name": step.name,
-                "executable": step.executable,
-                "argList": step.arglist,
+                "name": forward_model.name,
+                "executable": forward_model.executable,
+                "argList": forward_model.arglist,
             }
-            for step in steps
+            for forward_model in realization.forward_models
         ],
         "run_id": "0",
         "ert_pid": "0",
-        "real_id": "0",
+        "real_id": str(realization.iens),
     }
-
-    with open(path / "jobs.json", "w") as f:
+    realization_run_path = Path(realization.run_arg.runpath)
+    realization_run_path.mkdir()
+    with open(realization_run_path / "jobs.json", mode="w", encoding="utf-8") as f:
         json.dump(jobs, f)
 
 
 @pytest.fixture
 def realization(storage, tmp_path):
     ensemble = storage.create_experiment().create_ensemble(name="foo", ensemble_size=1)
+    return create_stub_realization(ensemble, tmp_path, 0)
+
 
+def create_stub_realization(ensemble, base_path: Path, iens) -> Realization:
     run_arg = RunArg(
         run_id="",
         ensemble_storage=ensemble,
-        iens=0,
+        iens=iens,
         itr=0,
-        runpath=str(tmp_path),
+        runpath=str(base_path / f"realization-{iens}"),
         job_name="",
     )
 
-    return Realization(
-        iens=0,
+    realization = Realization(
+        iens=iens,
         forward_models=[],
         active=True,
         max_runtime=None,
```

Comment:
If we want to keep it "general" shall max_runtime be a parameter too?

Reply:
There are currently no tests that use max_runtime, so I think it is better to add that later.

Reply:
We came to an agreement to skip this